storybook/webpack.config.js

```diff
@@ -42,6 +44,7 @@ module.exports = ( { config: storybookConfig } ) => {
 		'wordpress-components': require.resolve(
 			'../node_modules/wordpress-components'
 		),
+		'@emotion/styled': pathToModule( '../node_modules/@emotion/styled' ),
```

Comment:
I'm curious to know what the difference is between this line and 

```
'@emotion/styled': require.resolve('../node_modules/@emotion/styled'),
```

Reply:
Resurrected once again! 😬 

The difference is that `require.resolve` resolves the path to the CJS module which is exposed by the package, which would still cause problems. This is basically just outputting the string to the actual directory.

Reply:
Hmm, but if we pass the same path (`'../node_modules/@emotion/styled'`) to `require.resolve`, I would expect it to output the path to THAT particular node module, not the one resolved by default (i.e. the one inside the `emotion/styled` folder)? What am I missing? 

From the docs, 

> `require.resolve` Use(s) the internal require() machinery to look up the location of a module, but rather than loading the module, just return the resolved filename.

Reply:
Yea the module is the one you would have expected with `require.resolve`, but I believe it parses the package.json for the entry point or something like that. At any rate, it was linking me to the cjs bundle, but I literally just needed the path.